### PR TITLE
Fixed folijet tests

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1392/allure/#suites
***

### CODE REVIEW

File: cypress/e2e/data-import/e2e/mrc-import-file-for-modify-marcBib-record.cy.js [hunk 1]
```diff
@@ -51,8 +51,6 @@ describe('ui-data-import: Verify the possibility to modify MARC Bibliographic re
   });
 
   after(() => {
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/e2e/mrc-import-file-for-modify-marcBib-record.cy.js:53]: fix linter issues

File: cypress/support/fragments/data_import/dataImport.js [hunk 2]
```diff
@@ -192,20 +192,6 @@ export default {
...
-    cy.then(() => DataImportUploadFile().isDeleteFilesButtonExists()).then(isDeleteFilesButtonExists => {
-      if (isDeleteFilesButtonExists) {
-        cy.do(Button('Delete files').click());
-        cy.expect(orChooseFilesButton.exists());
-        cy.allure().endStep();
-      }
-    });
-  },
-
   checkIsLandingPageOpened: () => {
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/dataImport.js:195]: Fix linter issues

File: cypress/e2e/data-import/e2e/edi-import-file-for-create-invoice-record.cy.js [hunk 3]
```diff
@@ -34,19 +34,16 @@ describe('ui-data-import: EDIFACT file import with creating of new invoice recor
...
-    DataImport.checkUploadState();
-    Users.deleteViaApi(user.userId);
-
     // clean up generated profiles
     JobProfiles.deleteJobProfile(jobProfileName);
     ActionProfiles.deleteActionProfile(actionProfileName);
     FieldMappingProfiles.deleteFieldMappingProfile(mappingProfileName);
+    Users.deleteViaApi(user.userId);
+    cy.getInvoiceIdApi({ query: `vendorInvoiceNo="${FileDetails.invoiceNumberFromEdifactFile}"` })
+      .then(id => cy.deleteInvoiceFromStorageApi(id));
```
Comment (penfriend) [inline cypress/e2e/data-import/e2e/edi-import-file-for-create-invoice-record.cy.js:46]: deleteInvoiceFromStorageApi => deleteInvoiceFromStorageViaApi

File: cypress/e2e/data-import/e2e/edi-import-file-for-create-invoice-record.cy.js [hunk 4]
```diff
@@ -34,19 +34,16 @@ describe('ui-data-import: EDIFACT file import with creating of new invoice recor
...
-      .then(id => cy.deleteInvoiceFromStorageApi(id));
-    DataImport.checkUploadState();
-    Users.deleteViaApi(user.userId);
-
     // clean up generated profiles
     JobProfiles.deleteJobProfile(jobProfileName);
     ActionProfiles.deleteActionProfile(actionProfileName);
     FieldMappingProfiles.deleteFieldMappingProfile(mappingProfileName);
+    Users.deleteViaApi(user.userId);
+    cy.getInvoiceIdApi({ query: `vendorInvoiceNo="${FileDetails.invoiceNumberFromEdifactFile}"` })
```
Comment (penfriend) [inline cypress/e2e/data-import/e2e/edi-import-file-for-create-invoice-record.cy.js:45]: getInvoiceIdApi => getInvoiceIdViaApi

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/data-import/e2e/mrc-import-file-for-modify-marcBib-record.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/mrc-import-file-for-modify-marcBib-record.cy.js:81]: Done

File: cypress/e2e/data-import/e2e/edi-import-file-for-create-invoice-record.cy.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/edi-import-file-for-create-invoice-record.cy.js:45]: Done

**Pull request merged**